src/blockchain_db/sqlite/db_sqlite.h

```diff
@@ -74,7 +78,12 @@ class BlockchainSQLite : public db::Database
   // get_payments -> passing a block height will return an array of payments that should be created in a coinbase transaction on that block given the current batching DB state.
   std::optional<std::vector<cryptonote::batch_sn_payment>> get_sn_payments(uint64_t block_height);
 
-  // calculate_rewards -> takes the list of contributors from sn_info with their SN contribution amounts and will calculate how much of the block rewards should be the allocated to the contributors. The function will return a list suitable for passing to add_sn_payments
+  // calculate_rewards -> takes the list of contributors from sn_info with their SN contribution
+  // amounts and will calculate how much of the block rewards should be the allocated to the
+  // contributors. The function will return a list suitable for passing to add_sn_payments
+  //
+  // Note that distribution_amount here is typically passed as milli-atomic OXEN for extra
```

Comment:
`s/milli-atomic/milli-OXEN`

atomic unit is by definition the smallest amount possible.